schedview/app/prenight_inventory.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import datetime
+import os
+
+import pandas as pd
+from rubin_sim.sim_archive import read_archived_sim_metadata
+
+from schedview.compute import munge_sim_archive_metadata
+from schedview.dayobs import DayObs
+
+SIM_TABLE_COLUMNS = (
+    "resource_url",
+    "sim_execution_date",
+    "sim_index",
+    "telescope",
+    "scheduler_version",
+    "opsim_config_version",
+    "opsim_config_script",
+    "tags",
+)
+
+
+def get_prenight_table(
+    day_obs: DayObs,
+    archive_uri="s3://rubin:rubin-scheduler-prenight/opsim/",
+    num_nights=7,
```

Comment:
if num_nights the number of nights after day_obs to look for simulations to cover, or before? or is it maybe irrelevant?

Reply:
Before. I'll add this to the docstrinc.